data/consent-page.js

```diff
@@ -0,0 +1,66 @@
+const yo = require('yo-yo');
+
+// Build the study header
+const buildStudyTitle = studyName => {
+  const titleText = studyName || 'Participate in a new Firefox Study';
+  return yo`<h1>${titleText}</h1>`;
+};
+
+// Build the opt-in button
+const buildParticipateButton = (buttonText, buttonClasses) => {
+  const text = buttonText || `Try ${self.options.addonName}`;
+  return yo`
+  <button id="install-addon" class=${buttonClasses || 'primary'}>
+    ${text}
+  </button>`;
+};
+
+// Build the list of authors
+const buildAuthorsList = authors => {
+  const authorsArray = authors.split(', ');
+  return yo`<ul>
+    ${authorsArray.map(function (author) {
+      return yo`<li>${author}</li>`;
+    })}
+  </ul>`;
+};
+
+
+// Create studyTitle, button, and authorsList
+const studyTitle = buildStudyTitle(self.options.studyName);
+const participateButton = buildParticipateButton(self.options.buttonText);
+const authorsList = buildAuthorsList(self.options.authors);
+
+
+// Append header & button
+const headerElement = document.getElementById('study-header');
+      headerElement.appendChild(studyTitle);
+      headerElement.appendChild(participateButton);
```

Comment:
I ran lints, I think they showed up as warnings rather than errors and I had left them be.  Fixed.